scripts/appcenter-download-release.sh

```diff
@@ -97,6 +99,11 @@ function android() {
         "${APPCENTER_API_URL}/apps/pass-Culture/${appName}/releases" |
         jq 'map(select(.short_version == "'"${TARGET_VERSION}"'").id)[0]'
     )" || die "Cannot retrieve AppCenter Release ID"
+
+    if [[ "${releaseId}" == null ]]; then
```

Comment:
en bash, `null` n'est pas un mot clé du langage

pour enlever un risque de confusion, je recommande d'expliciter le fait que c'est une string

```suggestion
    if [[ "${releaseId}" == "null" ]]; then
```

Reply:
✅